### PR TITLE
default enabled for tls for kafka

### PR DESCRIPTION
**Pull request by JasonSinclair95**

### CODE REVIEW

**Review by davidcurrie — APPROVED** [review]
LGTM - don't merge until you have all the other pieces lined up though

**Comment by JasonSinclair95** [comment]
> PS - you've hit an issue where Helm doesn't like the `/` in the branch name. You could fix that by swapping out characters that are not supported in a chart version here: https://github.com/corda/corda-dev-helm/blob/main/Jenkinsfile#L6

Interesting. helm never complained about this version syntax before @davidcurrie

**Comment by davidcurrie** [comment]
> Interesting. helm never complained about this version syntax before @davidcurrie
`corda-runtime-os` doesn't have the same problem as it never uses the branch name. I have seen it before in this repo.

File: Jenkinsfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ def chartVersion() {
     if (env.BRANCH_NAME == 'main') {
         return "$chartVersion"
     } else {
-        return "$chartVersion-${env.BRANCH_NAME}"
+        return "$chartVersion/${env.BRANCH_NAME}"
```
[the remark this answers is not in the document]
Comment (JasonSinclair95) [reply Jenkinsfile]: okay thanks

**Pull request merged**